platform/default/online_file_source.cpp

```diff
@@ -117,6 +120,15 @@ class OnlineFileSource::Impl {
         pendingRequestsMap.erase(request);
 
         activateRequest(request);
+        assert(pendingRequestsMap.size() == pendingRequestsList.size());
+    }
+    
+    bool isScheduled(OnlineFileRequest* request) {
```

Comment:
Should be `isPending`. "Scheduled" means the timer is running -- it's a state that precedes "Pending".